backend/v3/storage/database/repository/settings.go

```diff
@@ -0,0 +1,1106 @@
+	return db_json.NewFieldChange([]string{"allowDomainDiscovery"}, value)
+}
+
+func (l loginSettings) SetDisableLoginWithEmailField(value bool) db_json.JsonUpdate {
+	return db_json.NewFieldChange([]string{"disableLoginWithEmail"}, value)
+}
+
+func (l loginSettings) SetDisableLoginWithPhoneField(value bool) db_json.JsonUpdate {
+	return db_json.NewFieldChange([]string{"disableLoginWithPhone"}, value)
+}
+
+func (l loginSettings) SetPasswordlessTypeField(value domain.PasswordlessType) db_json.JsonUpdate {
+	return db_json.NewFieldChange([]string{"passwordlessType"}, value)
+}
+
+func (l loginSettings) SetDefaultRedirectURIField(value string) db_json.JsonUpdate {
+	return db_json.NewFieldChange([]string{"defaultRedirectUri"}, value)
+}
+
+func (l loginSettings) SetPasswordCheckLifetimeField(value time.Duration) db_json.JsonUpdate {
+	return db_json.NewFieldChange([]string{"passwordCheckLifetime"}, value)
+}
+
+func (l loginSettings) SetExternalLoginCheckLifetimeField(value time.Duration) db_json.JsonUpdate {
+	return db_json.NewFieldChange([]string{"externalLoginCheckLifetime"}, value)
+}
+
+func (l loginSettings) SetMFAInitSkipLifetimeField(value time.Duration) db_json.JsonUpdate {
+	return db_json.NewFieldChange([]string{"mfaInitSkipLifetime"}, value)
+}
+
+func (l loginSettings) SetSecondFactorCheckLifetimeField(value time.Duration) db_json.JsonUpdate {
+	return db_json.NewFieldChange([]string{"secondFactorCheckLifetime"}, value)
+}
+
+func (l loginSettings) SetMultiFactorCheckLifetimeField(value time.Duration) db_json.JsonUpdate {
+	return db_json.NewFieldChange([]string{"multiFactorCheckLifetime"}, value)
+}
+
+func (l loginSettings) AddMFAType(value domain.MultiFactorType) db_json.JsonUpdate {
+	return db_json.NewArrayChange([]string{"mfaType"}, value, false)
+}
+
+func (l loginSettings) RemoveMFAType(value domain.MultiFactorType) db_json.JsonUpdate {
+	return db_json.NewArrayChange([]string{"mfaType"}, value, true)
+}
+
+func (l loginSettings) AddSecondFactorTypesField(value domain.SecondFactorType) db_json.JsonUpdate {
+	return db_json.NewArrayChange([]string{"secondFactors"}, value, false)
+}
+
+func (l loginSettings) RemoveSecondFactorTypesField(value domain.SecondFactorType) db_json.JsonUpdate {
+	return db_json.NewArrayChange([]string{"secondFactors"}, value, true)
+}
+
+func LoginRepository() domain.LoginRepository {
+	return &loginSettings{
+		&settings{},
+	}
+}
+
+var _ domain.LoginRepository = (*loginSettings)(nil)
+
+func (s *loginSettings) Set(ctx context.Context, client database.QueryExecutor, setting *domain.LoginSetting) error {
+	if setting == nil {
+		return ErrSettingObjectMustNotBeNil
+	}
+	setting.Type = domain.SettingTypeLockout
+	return createSetting(ctx, client, setting.Setting, setting)
+}
+
+func (s *loginSettings) Get(ctx context.Context, client database.QueryExecutor, opts ...database.QueryOption) (*domain.LoginSetting, error) {
+	loginSetting := &domain.LoginSetting{}
+	var err error
```

Comment:
You can name the return parameters, then you don't have to declare a variable here.